back/engines/commercial/public_api/app/serializers/public_api/v2/basket_idea_serializer.rb

```diff
@@ -5,6 +5,10 @@ module V2
     class BasketIdeaSerializer < PublicApi::V2::BaseSerializer
       type :basket_ideas
       attributes(:basket_id, :idea_id, :votes, :created_at, :updated_at)
+
+      attribute(:phase_id) { object.basket&.phase_id }
+      attribute(:project_id) { object.basket&.phase&.project_id }
+      attribute(:user_id) { object.basket&.user_id }
```

Comment:
Aren't those associations expected to be present?
```suggestion
      attribute(:phase_id) { object.basket.phase_id }
      attribute(:project_id) { object.basket.phase.project_id }
      attribute(:user_id) { object.basket.user_id }
```

Reply:
Yes indeed they are